<ui version="4.0" >
 <class>createNetworkGameDialog</class>
 <widget class="QDialog" name="createNetworkGameDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Create Network Game</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Network Game Settings</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="6" column="1" colspan="3" >
       <widget class="QLineEdit" name="lineEdit_serverPassword" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Server Password:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="4" >
       <widget class="Line" name="line" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3" >
       <widget class="QSpinBox" name="spinBox_quantityPlayers" >
        <property name="maximum" >
         <number>5</number>
        </property>
        <property name="minimum" >
         <number>2</number>
        </property>
        <property name="value" >
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3" >
       <widget class="QSpinBox" name="spinBox_startCash" >
        <property name="maximum" >
         <number>5000</number>
        </property>
        <property name="minimum" >
         <number>1000</number>
        </property>
        <property name="value" >
         <number>2000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3" >
       <widget class="QSpinBox" name="spinBox_smallBlind" >
        <property name="maximum" >
         <number>500</number>
        </property>
        <property name="minimum" >
         <number>5</number>
        </property>
        <property name="value" >
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Game Speed for Computer Opponent:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Number of Players:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Small Blind:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Start Cash:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Hands before raise Small Blind:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2" >
       <widget class="QSpinBox" name="spinBox_handsBeforeRaiseSmallBlind" >
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="4" column="3" >
       <widget class="QSpinBox" name="spinBox_gameSpeed" >
        <property name="maximum" >
         <number>11</number>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>4</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>181</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_createGame" >
       <property name="text" >
        <string>Create Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBox_quantityPlayers</tabstop>
  <tabstop>spinBox_startCash</tabstop>
  <tabstop>spinBox_smallBlind</tabstop>
  <tabstop>spinBox_handsBeforeRaiseSmallBlind</tabstop>
  <tabstop>spinBox_gameSpeed</tabstop>
  <tabstop>pushButton_createGame</tabstop>
  <tabstop>pushButton_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_createGame</sender>
   <signal>clicked()</signal>
   <receiver>createNetworkGameDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>168</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel" >
     <x>87</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>createNetworkGameDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>284</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel" >
     <x>311</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
